src/components/Physics.vue

```diff
@@ -1,11 +1,13 @@
 <script setup lang="ts">
-import { useLoop } from '@tresjs/core'
+import { EventQueue } from '@dimforge/rapier3d-compat'
+import { useLoop, useTresContext } from '@tresjs/core'
 import { Vector3 } from 'three'
 import { watch } from 'vue'
 import type { VectorCoordinates } from '@tresjs/core'
 import { useRapierContextProvider } from '../composables/useRapier'
-
 import { GRAVITY } from '../constants/physics'
+
+import { collisionEmisor, get3DGroupFromSource, getSourceFromColliderHandle } from '../utils/collisions'
```

Comment:
Normally we should be able to import things directly from a specific folder
Here :

- `import { GRAVITY } from '../constants'`
- `import { collisionEmisor, get3DGroupFromSource, getSourceFromColliderHandle } from '../utils'`

Reply:
Done